### PR TITLE
Add preliminary support for the ROG Crosshair X670E Hero.

### LINKED ISSUE #39: Add support for Crosshair X670E Hero
I just got a X670E Hero and have added support for the first few temperatures that were easy to validate.  There appears to be more in the EC on these boards, but I've at least nailed down some of the basic component temps as well as the water temperature sensors.

I'm attaching the DSDT dump to this issue and will create a pull request shortly.

### PR DESCRIPTION
**Pull request by kyndigen**
Only the temp sensors that I can verify are present.  HWINFO in Windows shows other accumulated data and statistics (time since installed, total power used, etc) that I have not attempted to find.

Resolves #39.

### CODE REVIEW

File: asus-ec-sensors.c [hunk 1]
```diff
@@ -101,6 +101,8 @@ enum ec_sensors {
 	ec_sensor_temp_chipset,
 	/* CPU temperature [℃] */
 	ec_sensor_temp_cpu,
+	/* CPU package temperature [℃] */
+	ec_sensor_temp_package,
```
Comment (zeule) [inline asus-ec-sensors.c:105]: `ec_sensor_temp_cpu_package`, please.

File: asus-ec-sensors.c [hunk 2]
```diff
@@ -139,6 +141,7 @@ enum ec_sensors {
 
 #define SENSOR_TEMP_CHIPSET BIT(ec_sensor_temp_chipset)
 #define SENSOR_TEMP_CPU BIT(ec_sensor_temp_cpu)
+#define SENSOR_TEMP_PACKAGE BIT(ec_sensor_temp_package)
```
Comment (zeule) [inline asus-ec-sensors.c:144]: Likewise, `SENSOR_TEMP_CPU_PACKAGE`.

**Review by zeule — CHANGES_REQUESTED** [review]
Thank you! Could you make one little change to naming, please, to make the new sensor name precise?

**Review by zeule — APPROVED** [review]

**Comment by zeule** [comment]
Thank you! Do you plan to look for those sensors you mentioned in the near future, or shall I submit this upstream as is? 6.4 merge window is still open, AFAIK.

**Comment by kyndigen** [comment]
Go ahead and upstream it as-is.  I'm not going to have much bandwidth to do further digging for a while.

**Pull request merged**